tests/rptest/tests/audit_log_tests.py

```diff
@@ -174,3 +177,46 @@ def combine(acc, x):
             call_apis()
         self.logger.debug("Finished 500 api calls with management disabled")
         _ = number_of_records_matching(api_keys, 1000)
+
+    @cluster(num_nodes=3)
+    def test_audit_log_metrics(self):
```

Comment:
One note here about the tests, when I created this file I in correctly named it `audit_log_tests.py` and therefore CI is not running it. It should be named `audit_log_test.py`.

Reply:
So you can introduce a seperate commit to change the name, then make your changes, you may have to resolve conflicts with Mike as he is doing the same for #14452 if he merges his PR before this one

Reply:
Ah, I hadn't noticed either. Thanks for the heads up.